solana-programs/staking-bridge/README.md

```diff
@@ -0,0 +1,66 @@
+# Staking Bridge
+
+This program has 2 main functions:
+1. To swap SOL USDC tokens to SOL AUDIO tokens via the Raydium AMM Program.
+2. To convert SOL AUDIO tokens to ETH AUDIO tokens via the Wormhole Token Bridge.
+
+The program has a PDA (Program Derived Address) account that will own the tokens.
+The program methods are permissionless, meaning that anyone can interact with them as long as they're willing to pay for the transactions.
+
+Please note that this program and its Anchor tests are set up to only work with the Solana mainnet cluster at the moment.
+
+## Building the project
+Make sure you have `anchor` installed. See https://www.anchor-lang.com/docs/installation. If you follow the installation steps, you should also have Rust, Solana, and Yarn installed.
+
+Build the project.
+```
+anchor build
+```
+
+Anchor uses Yarn, so please sure you have it installed, and run
+```
+yarn install
+```
+
+Note that I've been using Node 16 for this project, as I was running into issues with older Node versions for the `@solana/spl-token` and `@solana/web3.js` modules.
```

Comment:
let's add an nvmrc instead of this comment :) 

see https://github.com/AudiusProject/audius-client/blob/main/.nvmrc

Also, can you test node 18? we should support that. @sliptype is moving everything to 18 right now.

Reply:
will do